src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsExpensesFromPage.tsx

```diff
@@ -191,19 +190,20 @@ function WorkspaceWorkflowsApprovalsExpensesFromPage({policy, isLoadingReportDat
                         onBackButtonPress={goBack}
                     />
                     {approvalWorkflow?.action === CONST.APPROVAL_WORKFLOW.ACTION.CREATE && (
-                        <Text style={[styles.textHeadlineH1, styles.mh5, styles.mv3]}>{translate('workflowsExpensesFromPage.header')}</Text>
+                        <Text style={[styles.textHeadlineH1, styles.mh5, styles.mv3]}>
+                            {translate(isMembersListEmpty ? 'workflowsExpensesFromPage.emptyMembersList' : 'workflowsExpensesFromPage.header')}
+                        </Text>
                     )}
                     <SelectionList
                         canSelectMultiple
                         sections={sections}
                         ListItem={InviteMemberListItem}
-                        textInputLabel={translate('selectionList.findMember')}
+                        textInputLabel={isMembersListEmpty ? undefined : translate('selectionList.findMember')}
```

Comment:
```suggestion
                        textInputLabel={!searchTerm && isMembersListEmpty ? undefined : translate('selectionList.findMember')}
```

We need to keep input when there are no search results

https://github.com/user-attachments/assets/99cdb937-f362-42e2-af83-13d72c4d7149